packages/rocketchat-api/server/v1/users.js

```diff
@@ -26,9 +26,9 @@ RocketChat.API.v1.addRoute('users.create', { authRequired: true }, {
 				RocketChat.saveCustomFields(newUserId, this.bodyParams.customFields);
 			}
 
-			if (this.bodyParams.active === false) {
+			if (typeof this.bodyParams.data.active !== 'undefined') {
```

Comment:
From where are you getting this `data` object inside `bodyParams`? It's not in our check list above.